libfreerdp/core/transport.c

```diff
@@ -470,6 +471,10 @@ BOOL transport_connect(rdpTransport* transport, const char* hostname, UINT16 por
 
 	if (transport->GatewayEnabled)
 	{
+		if (settings->GatewayArmTransport)
```

Comment:
for new code better use `freerdp_settings_get_bool` (and no braces for one liners)